packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -88,13 +88,15 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
       return openCreateOrganization({ afterCreateOrganizationUrl, skipInvitationScreen });
     };
 
-    const handleManageOrganizationClicked = () => {
+    const handleManageOrganizationClicked = (path?: string) => {
```

Comment:
🙃
Rename the function to something more generic, and either use that directly for the onClick props, or simply have `handleManageOrganizationClicked = () => handleItemClick()`